ios/brave-ios/Sources/AIChat/Components/Feedback/AIChatFeedbackToastView.swift

```diff
@@ -0,0 +1,160 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import DesignSystem
+
+private struct AIChatFeedbackToastModifier: ViewModifier {
+  @State
+  private var task: Task<Void, Error>?
+  
+  private let displayDuration = 5.0
+
+  @Binding
+  var toastType: AIChatFeedbackToastType
+  
+  func body(content: Content) -> some View {
+    content
+      .frame(maxWidth: .infinity, maxHeight: .infinity)
+      .overlay(
+        ZStack {
+          if toastType != .none {
+            VStack {
+              Spacer()
+              AIChatFeedbackToastView(toastType: $toastType)
+            }
+            .transition(.move(edge: .bottom))
+            .offset(y: -10.0)
+          }
+        }
+          .animation(.spring(), value: toastType)
```

Comment:
You may be able to dump ZStack and place this animation on the transition itself e.g.

`.transition(.move(edge: .bottom)).animation(.spring())`